examples/functions/fares/Netex_101.21_TfL_GeographicFares_UnitZone_MultipleProduct.xml

```diff
@@ -3520,9 +3520,9 @@ Oyster_PayAsYouGo_right -   PayAsYouGo_trip
 										<VehicleModes>metro</VehicleModes>
 									</validityParameters>
 								</GenericParameterAssignment>
-								<GenericParameterAssignment version="any" order="2" id="tfl:Prepaid_trip">
+								<GenericParameterAssignment version="any" order="2" id="tfl:Prepaid_trip@single">
 									<Name>Cant Break Metro journeys</Name>
-									<Description>  														Flat far simgle tripcase. </Description>
+									<Description>Flat fare single trip case.</Description>
```

Comment:
Was this already broken?

Reply:
Example currently validates in CEN repo as xsd:key constraint check for ValidityParameterAssignment_AnyVersionedKey (i.e. including GenericParameterAssignment IDs) is missing in the CEN master NeTEx_publication.xsd file and therefore this specific check was not triggered i.e. no fail in Travis xmllint.

This - and further - improvements will be PR'ed from Entur when we have finalized / reached agreement on related enhancements. 

As the local build highlighted these minor bugs, I updated the example file. (Only one with this flaw found so far.)

Reply:
No I mean the 'empty' description?

Reply:
Ignore my stupidity I didn't notice the many spaces.